sssom/util.py

```diff
@@ -1542,6 +1542,8 @@ def invert_mappings(
     :param predicate_invert_dictionary: YAML file providing the inverse mapping for predicates.
     :return: Pandas dataframe with all subject IDs having the same prefix.
     """
+    if PREDICATE_MODIFIER not in df.columns:
+        df["predicate_modifier"] = ""
```

Comment:
Remind me why this has to be here? Will the column removed if this turns out empty?

Reply:
Will the column appear in the inverted data frame even if its empty?

Reply:
for now , yes. But sometimes even if the column exists aren't the chances of it being completely empty high?

Reply:
I would prefer if the column is not added if it didn't exist earlier. Is this difficult to achieve?

Reply:
Fixed.